docs/introduction/en/basics.md

```diff
@@ -0,0 +1,109 @@
+---
+id: conflux_basics
+title: Basics
+custom_edit_url: https://github.com/Conflux-Chain/conflux-developer-site/edit/master/docs/introduction/en/basics.md
+keywords:
+  - conflux
+  - basics
+---
+In an overview, the Conflux world-state consists of a list of accounts and the associated account states, and the global state is updated via transactions. The Conflux blockchain stores all processed transactions in blocks, together with necessary information in block headers which enables a total ordering of all blocks. Below are some concepts you should to know.
+
+## CFX
+To incentivize the maintenance of the Conflux network and charge users for consumption of resources, Conflux has an intrinsic currency called Conflux Coin or simply Conflux, denoted by CFX for short. The smallest subdenomination is denoted by Drip, in which all values processed in Conflux are integers. One Conflux is defined as 10^18 Drip. Frequently used subdenominations of Conflux are list as follows:
+
+| Multiplier (in Drip) | Name   |
+| ------------- |:-------------:|
+| 10^0          | Drip          |
+| 10^9          | GDrip         |
+| 10^12         | uCFX          |
+| 10^18         | Conflux(CFX)  |
+
+## chainId & networkId
+`chainId` is a binary id indicating where T is intended to be executed. Which is used to prevent transaction replay attack.
```

Comment:
What is "T"? Use "transaction".
`chainId` is a binary indicating where a transaction is intended to be executed. It is used to prevent transaction replay attack.